pkg/rpc/response/result/validator.go

```diff
@@ -1,14 +1,13 @@
 package result
 
 import (
-	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
 	"github.com/nspcc-dev/neo-go/pkg/util"
 )
 
 // Validator used for the representation of
 // state.Validator on the RPC Server.
 type Validator struct {
-	PublicKey keys.PublicKey `json:"publickey"`
-	Votes     util.Fixed8    `json:"votes"`
-	Active    bool           `json:"active"`
+	PublicKey string      `json:"publickey"`
```

Comment:
I'd prefer to keep it `keys.PublicKey` making hexification transparent. We don't yet have JSON marshalling defined for keys, so maybe it's time to add it.